server/src-lib/Hasura/Server/Query.hs

```diff
@@ -290,6 +295,17 @@ queryNeedsReload (RQV2 qi) = case qi of
   RQV2TrackTable _           -> True
   RQV2SetTableCustomFields _ -> True
 
+-- TODO: add RQSelect after console changes
```

Comment:
What does this `TODO` comment mean?

Reply:
The RQL select statement should also be(ideally) run in READ ONLY mode. But this could be part of the current console's bulk statement and hence would be breaking if we do it now itself.